### ISSUE TITLE
I save the the model,(h5->pb)but some FusedBatchNorm(is_training =ture) ,how to fix it to  False

### ISSUE THREAD

**Issue by ptklx** [qubvel/segmentation_models#223]
![Screenshot from 2019-10-11 09-58-56](https://user-images.githubusercontent.com/18067178/66618972-cd068f80-ec0d-11e9-82d6-88598eccf087.png)
![Screenshot from 2019-10-11 10-00-20](https://user-images.githubusercontent.com/18067178/66619021-f6bfb680-ec0d-11e9-9ec9-4280dde0c226.png)

when the FusedBthcNorm(is_training =ture),the mean_value and  the var_value is NULL,and inference is not right,

I find some  use tf.control_dependencies and tf.GraphKeys.UPDATE_OPS fix BatchNorm bug

**Comment by qubvel** [comment]
May be `keras.backend.learning_phase(0)` before loading h5 model will help you

**Comment by ptklx** [comment]
very thank you , add this it's work
